targets/targets.json

```diff
@@ -2426,24 +2426,25 @@
         "features": ["BLE"],
         "release_versions": ["2", "5"]
     },
-    "RZ_A1H": {
-        "supported_form_factors": ["ARDUINO"],
+    "RZ_A1XX": {
+        "inherits": ["Target"],
         "core": "Cortex-A9",
-        "program_cycle_s": 2,
-        "extra_labels": ["RENESAS", "RZ_A1XX", "RZA1H", "MBRZA1H"],
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
-        "inherits": ["Target"],
+        "extra_labels": ["RENESAS", "RZ_A1XX"],
+        "program_cycle_s": 2
+    },
+    "RZ_A1H": {
+        "inherits": ["RZ_A1XX"],
+        "supported_form_factors": ["ARDUINO"],
+        "extra_labels_add": ["RZA1H", "MBRZA1H"],
         "device_has": ["ANALOGIN", "CAN", "ETHERNET", "I2C", "I2CSLAVE", "I2C_ASYNCH", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_ASYNCH", "SERIAL_FC", "SPI", "SPISLAVE", "SPI_ASYNCH", "STDIO_MESSAGES"],
```

Comment:
small improvement - if RZ_A1XX is   the parent, that one contains the implementation for entire HAL, therefore device_has can be moved there. Each inherited target can overwrite it add/remove.